tools/wltests/test_series

```diff
@@ -305,6 +304,13 @@ if [ "x$DEVICE" == "xC00010FFBAADA555" ]; then
 	fi
 	exit $EINVAL
 fi
+export WA_LOCAL_CONF=$RESULTS/config_local.yaml
```

Comment:
This is done before the $RESULT dir is created. As a result, the file creation fails, and the `device_config` option never gets written in local_config.yaml.

Reply:
:hankey:

Reply:
You probably just need to move that hunk below the `# Prepare RESULTS` hunk though :smile:

Reply:
yes... sorry... hope this is the :cake: time...